### ISSUE TITLE
Endless loop of provisioning when deploying Unifi Controller with helm chart.

### ISSUE THREAD

**Issue by dniel** [billimek/billimek-charts#43]
Hi @billimek just a couple of question, 

1) have you had trouble with endless loops of re-provisioning config with the unifi controller when deployed in kubernetes? I tried your chart but I ended up in a endless loop and re-provisioned all devices until I stopped the controller.

2) how do you handle STUN communication?  What kind of ingress controller do you use? is it necessary for the unifi controller to have a working STUN communication? Im using Traefik and it doesnt support UDP traffic so I cant expose the STUN port.

**Comment by billimek** [comment]
Hi @dniel,

#1 is an interesting problem and I can't say that I've ever experienced this.  You can initially 'see' the device from the controller and when attempting to provision them, they get stuck in a provision loop?  I wonder if there is a missing 'path' for the devices to call-back to the controller?  Do you know if the controller service (the one that defaults to listening on port 8080) is reachable/visible to the devices?

Regarding #2, I personally handle the STUN communication via a LoadBalancer (specifically [metalLB](https://metallb.universe.tf/)) to get a static IP address and then use my router (edgerouter) to forward requests for UDP/3478 to the stub service loadbalancer IP address.  From what I remember, having the STUN service wired-up is only a 'nice-to-have' and doesn't impact the operation of the controller, but I didn't like seeing the yellow warning symbol so wired-it up anyway.

For a point of comparison, this is the configuration I use for the unifi chart: https://github.com/billimek/k8s-gitops/blob/master/deployments/unifi.yaml (and [this](https://github.com/billimek/k8s-gitops/blob/master/setup/manual-steps/values-to-encrypt/unifi-values.txt) for ingress hostname config)

**Comment by dniel** [comment]
Yes, all seems ok I log in to my devices to set the inform address to the new controller running in kubernetes and it seems to work fine before starting to pushing config to the devices endlessly. And I see my Access Points continuously restart, and I lose connection when they are down. So there is definitely connection between the devices and the controller (and the status when running the info command on the devices reports correct adopted and OK inform url).

I have to try to debug. Could be resource starvation that make the controller to miss behave.

**Comment by dniel** [comment]
@billimek I think the trouble with re-provisioning loop is due to running the controller in Amazon EKS, which has a load balancer that is automatically scaled between zones to provide HA. Seems like the controller finds out that the ip address of changes when changing availability zone and issue a re-provisioning for each external ip-change. 

In the controller logs it look like this.
` inform - inform url or ip changed (https://ubnt.xx.xx/inform/<ip> -> https://ubnt.dniel.se/inform/<ip>), update mgmt`
